Simplenote/gradle.properties-example

```diff
@@ -0,0 +1,5 @@
+simperiumAppId=FIXME
+simperiumAppKey=FIXME
+hockeyAppId=FIXME
+googleAnalyticsId=FIXME
+facebookAppId=FIXME
```

Comment:
We should update the `README` to rename this file maybe? Doesn't need to be in this PR